FCCSW_ecal/run_thetamodulemerged.py

```diff
@@ -4,7 +4,9 @@
 from Configurables import PodioOutput
 from Configurables import CorrectCaloClusters
 from Configurables import CreateCaloClustersSlidingWindow
```

Comment:
if CreateCaloClustersSlidingWindow is not used anymore, can you remove this line?